src/utils/tokens.ts

```diff
@@ -264,7 +267,7 @@ const tokens = {
     });
 
     if (createAnotherMint) {
-      const newAddress = await storage.getCurrentAddress();
+      const newAddress = mintAuthorityAddress || await storage.getCurrentAddress();
```

Comment:
Should we validate that `mintAuthorityAddress` belongs to this wallet?

Reply:
Why is this resolved? Shouldn't we like we do it like we do with the other token actions?

Reply:
I resolved this comment because we decided to add another parameter to allow external addresses. There might be a situation where a use case wants to send the mint authority to another address, so we are allowing it but it must be explicitly enabled.